Qlc/Objects/RewardInfo.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Newtonsoft.Json;
+
+namespace Qlc.Objects
+{
+    public class RewardInfo
```

Comment:
change RewardInfo to MinerRewardInfo just for clear.

Reply:
Done